examples/applications/astradb-sink/README.md

```diff
@@ -21,16 +21,22 @@ CREATE TABLE IF NOT EXISTS products (
 
 This is handled by the 'cassandra-table' assets in the pipeline.yaml file.
 
-## Configure the pipeline
+## Configure access to the Vector Database
 
-Update the secrets.yaml file and set the Astra credentials and the database name:
-- clientId
-- secret
-- token
-- database
+Export some ENV variables in order to configure access to the database:
+
+```
+export ASTRA_TOKEN=...
```

Comment:
```suggestion
export ASTRA_TOKEN=AstraCS:xxxx
```

Reply:
I don't think it is worth, the user has to copy/paste from the Astra UI, and we are now using the same terms: token, secret, clientId. Previously we called them username/password with a custom mapping to clientId and secret